.changeset/pre.json

```diff
@@ -0,0 +1,54 @@
+{
+  "mode": "pre",
+  "tag": "alpha",
+  "initialVersions": {
+    "@contentful/f36-accordion": "4.68.1",
+    "@contentful/f36-asset": "4.68.1",
+    "@contentful/f36-autocomplete": "4.68.1",
+    "@contentful/f36-avatar": "4.68.1",
+    "@contentful/f36-badge": "4.68.1",
+    "@contentful/f36-button": "4.68.1",
+    "@contentful/f36-card": "4.68.1",
+    "@contentful/f36-collapse": "4.68.1",
+    "@contentful/f36-copybutton": "4.68.1",
+    "@contentful/f36-datepicker": "4.68.1",
+    "@contentful/f36-datetime": "4.68.1",
+    "@contentful/f36-drag-handle": "4.68.1",
+    "@contentful/f36-empty-state": "4.68.1",
+    "@contentful/f36-entity-list": "4.68.1",
+    "@contentful/f36-forms": "4.68.1",
+    "@contentful/f36-header": "4.68.1",
+    "@contentful/f36-icon": "5.0.0-alpha.20",
+    "@contentful/f36-icons": "5.0.0-alpha.31",
+    "@contentful/f36-image": "4.68.1",
+    "@contentful/f36-layout": "5.0.0-alpha.15",
+    "@contentful/f36-list": "4.68.1",
+    "@contentful/f36-menu": "4.68.1",
+    "@contentful/f36-modal": "4.68.1",
+    "@contentful/f36-multiselect": "4.24.0",
+    "@contentful/f36-navbar": "5.0.0-alpha.35",
+    "@contentful/f36-navlist": "4.1.0-alpha.1",
```

Comment:
This and the progress-stepper won't be bumped to the v5 when leaving the prerelease mode